src/Jackett.Common/Indexers/BJShare.cs

```diff
@@ -162,167 +182,32 @@ private string FixAbsoluteNumbering(string title)
                 title = Regex.Replace(title, @"(Ep[\.]?[ ]?)|([S]\d\d[Ee])", "");
                 return title;
             }
-            else if (title.Contains("[Novela]"))
+            if (title.Contains("[Novela]"))
             {
                 title = Regex.Replace(title, @"(Cap[\.]?[ ]?)", "S01E");
                 title = Regex.Replace(title, @"(\[Novela\]\ )", "");
                 title = Regex.Replace(title, @"(\ \-\s*Completo)", " - S01");
                 return title;
             }
-            else
-            {
-                return title;
-            }
+            return title;
         }
 
         protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuery query)
         {
             query = query.Clone(); // avoid modifing the original query
```

Comment:
@ngosang I don't think we do this in any other tracker. Should we be?

Reply:
Maybe it's needed because we change the original search term

Reply:
That would make sense as to why we do it here, but I recall seeing other trackers changing the search terms as well without closing the query. I'm also fairly certain that we don't use the query again after PerformQuery, so it wouldn't make a difference if we did change it.

But I could be wrong, and I'd like to see if @ngosang or maybe even @garfield69 know something I don't.

Reply:
I prefer to not modify the original `query` but I also recall to see it in some trackers. Since you are only changing the search term near to the usage, maybe we should remove the `.clone()` and create a nu var with the modified search term.

Reply:
Ok, the SearchTerm change was needed because of the MatchQueryStringAND called at end of every row, but saw now that it has an "queryStringOverride" parameter, so i refactored it so it would not be needed anymore to modify the original query object, hence the clone is not needed anymore.